wiki/Gameplay/Daily_challenge/de.md

```diff
@@ -0,0 +1,60 @@
+# Tägliche Herausforderung
+
+Die **tägliche Herausforderung** (auch *Daily Challenge*) ist ein Mehrspielermodus in [osu!(lazer)](/wiki/Client/Release_stream/Lazer), bei dem Spieler täglich in Folge immer schwieriger werdende Beatmaps abschließen müssen. Alle 7 Tage wird das Schwierigkeitsniveau zurückgesetzt.
+
+Für diesen Modus werden Beatmaps von einer Gruppe von Mitwirkenden ausgewählt, die hauptsächlich aus dem [Featured-Artist](/wiki/People/Featured_Artists)-Katalog stammen. An manchen Tagen werden möglicherweise Mods vorgegeben, mit denen die Beatmap abgeschlossen werden muss.
+
+## Spielmenü
+
+Das Menü für die tägliche Herausforderung kann vom Hauptmenü aus folgendermaßen erreicht werden:
+
+1. Klicke den Button `Spielen` oder drücke `P`.
+2. Klicke den Button `Tägliche Herausforderung` oder drücke `D`.
+
+![](img/daily-challenge-menu.png "Screenshot der täglichen Herausforderung am 7. November 2024.")
+
+Beim Öffnen der täglichen Herausforderung wird ein Intro angezeigt, das die abzuschließende Beatmap sowie die vorgegebenen Mods präsentiert.
```

Comment:
```suggestion
Beim erstmaligen Öffnen der täglichen Herausforderung wird ein Intro angezeigt, das die abzuschließende Beatmap sowie die vorgegebenen Mods präsentiert.
```

`for the first time` is not in the original article but maybe it makes sense to add this here and in the original article? 🤔

Reply:
It's fine either way IMO

Reply:
Will leave it out for now